doc/man/zcashd-wallet-tool.1

```diff
@@ -0,0 +1,54 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.8.
+.TH ZCASHD-WALLET-TOOL "1" "March 2022" "zcashd-wallet-tool v4.6.0-1-9d5762e47" "User Commands"
+.SH NAME
+zcashd-wallet-tool \- manual page for zcashd-wallet-tool v4.6.0-1-9d5762e47
+.SH SYNOPSIS
+.B zcashd-wallet-tool
+[\fI\,OPTIONS\/\fR]
+.SH DESCRIPTION
+.SS "Optional arguments:"
+.TP
+\fB\-\-help\fR
+Print this help output
+.TP
+\fB\-\-conf\fR FILENAME
+Specify configuration filename, relative to the data directory (default: zcash.conf)
+.TP
+\fB\-\-datadir\fR PATH
+Specify data directory (this path cannot use '~')
+.TP
+\fB\-\-testnet\fR
+Use the test chain
+.TP
+\fB\-\-rpcconnect\fR IPADDR
+Send commands to node running on IPADDR (default: 127.0.0.1)
+.TP
+\fB\-\-rpcport\fR PORT
+Connect to JSON\-RPC on PORT (default: 8232 or testnet: 18232)
+.TP
+\fB\-\-rpcuser\fR USERNAME
+Username for JSON\-RPC connections
+.TP
+\fB\-\-rpcpassword\fR PASSWORD
+Password for JSON\-RPC connections
+.TP
+\fB\-\-rpcclienttimeout\fR SECONDS
+Timeout in seconds during HTTP requests, or 0 for no timeout. (default: 900)
+.PP
+Options can be given in GNU style (`\-\-conf=CONF` or `\-\-conf CONF`),
+or in Bitcoin style with a single hyphen (`\-conf=CONF`).
+.PP
+The environment variable RUST_LOG controls debug output, e.g.
+`RUST_LOG=debug`.
+.SH COPYRIGHT
+
+In order to ensure you are adequately protecting your privacy when using Zcash,
+please see <https://z.cash/support/security/>.
+
+Copyright (C) 2009-2022 The Bitcoin Core Developers
```

Comment:
Non-blocking: This is part of the footer file, but doesn't actually apply to the wallet tool. It would be nice to strip out the inapplicable `Bitcoin Core Developers` line.